Recipes.Test/JsonConfiguration/MinimalJsonSerializerTest.cs

```diff
@@ -0,0 +1,137 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="MinimalJsonSerializerTest.cs">
+//     Copyright (c) 2016. All rights reserved. Licensed under the MIT license. See LICENSE file in
+//     the project root for full license information.
+// </copyright>
+// <auto-generated>
+// Sourced from NuGet package. Will be overwritten with package update except in Spritely.Recipes source.
+// </auto-generated>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Spritely.Recipes.Test
+{
+    using System.ComponentModel;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq;
+    using Newtonsoft.Json.Linq;
+
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class MinimalJsonSerializerTest
+    {
+        [Test]
+        public void SerializeObject_without_type_serializes_to_json_using_MinimalSerializerSettings()
+        {
+            var dog = new Dog(5, "spud", FurColor.Brindle);
```

Comment:
I am verifying minimal because the null Nickname (new property added to the Dog type) is not serialized